src/platform/linux.rs

```diff
@@ -824,22 +863,36 @@ mod desktop {
                 "ps -u {} -f | grep 'Xorg' | grep -v 'grep'",
                 &self.uid
             )) {
+                log::debug!("extracting xauth file for Xorg running with uid: {}", self.uid);
```

Comment:
Too many log::debug! which it will be print to file, We'd better remove these log::debug before code merge.

Reply:
Debug messages was removed now